### PR TITLE
Remove the {{link}} macro from Russian

### PR DESCRIPTION
**Pull request by MrBrain295**
Part of: #7566

### CODE REVIEW

File: files/ru/glossary/javascript/index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h3 id="Общие_сведения">Общие сведения</h3>
 <h3 id="Изучение_JavaScript">Изучение JavaScript</h3>
 
 <ul>
- <li>{{Link("/ru/docs/Web/JavaScript/Guide")}} на MDN</li>
+ <li>[Руководство JavaScript](/ru/docs/Web/JavaScript/Guide) на MDN</li>
```
Comment (yin1999) [inline files/ru/glossary/javascript/index.html:34]:
```suggestion
 <li><a href="/ru/docs/Web/JavaScript/Guide">Руководство JavaScript</a> на MDN</li>
```

File: files/ru/glossary/javascript/index.html [hunk 2]
```diff
@@ -41,6 +41,6 @@ <h3 id="Технический_справочник">Технический сп
 
 <ul>
  <li><a href="http://www.ecma-international.org/publications/standards/Ecma-262.htm" rel="external">Последний стандарт ECMAScript</a></li>
- <li>{{Link("/ru/docs/Web/JavaScript/reference")}} на MDN</li>
+ <li> [Справочник по JavaScript](/ru/docs/Web/JavaScript/reference) на MDN</li>
```
Comment (yin1999) [inline files/ru/glossary/javascript/index.html:44]:
```suggestion
 <li><a href="/ru/docs/Web/JavaScript/reference">Справочник по JavaScript</a> на MDN</li>
```

File: files/ru/glossary/json/index.html [hunk 3]
```diff
@@ -26,5 +26,5 @@ <h3 id="Общие_знания">Общие знания</h3>
 <h3 id="Техническая_информация">Техническая информация</h3>
 
 <ul>
- <li>{{Link("/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON")}} на MDN</li>
+ <li>[JSON](/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON) на MDN</li>
```
Comment (yin1999) [inline files/ru/glossary/json/index.html:29]:
```suggestion
 <li><a href="/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON">JSON</a> на MDN</li>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
Those are html file, so we should use `<a>` tag

**Comment by MrBrain295** [comment]
@sashasushko

**Review by yin1999 — APPROVED** [review]
LGTM

**Comment by yin1999** [comment]
@caugner shall we merge this?

**Pull request merged**